spec/motions-spec.coffee

```diff
@@ -1744,6 +1745,38 @@ describe "Motions", ->
       keydown(',')
       expect(editor.getCursorScreenPosition()).toEqual [0, 2]
 
+    it "gets state from GlobalVimState", ->
```

Comment:
I think you can avoid making reference to the implementation here: something like `it "shares the most recent find command with other editors", ->`